cli/cmd/compose/ps.go

```diff
@@ -102,3 +115,12 @@ func runPs(ctx context.Context, opts psOptions) error {
 		},
 		"NAME", "SERVICE", "STATUS", "PORTS")
 }
+
+func contains(slice []string, item string) bool {
```

Comment:
allready declared in ecs/context.go
as well as local/util.go
Those are just a few line, but many better to avoid more duplication
let's move those all into utils package, ok?